.stylelintrc.js

```diff
@@ -15,7 +15,7 @@
 
 module.exports = {
   customSyntax: 'postcss-scss',
-  extends: ['stylelint-config-standard', 'stylelint-config-prettier'],
```

Comment:
Should be on devkit (and what's the reason behind this change)

Reply:
the plugin for prettier is deprecated since last stylelint version (all rules related got deprecated in core, so it does not conflict anymore)